action/actctx.go

```diff
@@ -72,5 +78,9 @@ func (act *AbstractAction) SanityCheck() error {
 	if act.GasPrice().Sign() < 0 {
 		return errors.Wrap(ErrGasPrice, "negative value")
 	}
+	// Reject execution of chainID not equal the node's chainID
+	if act.ChainID() != config.ChainID() {
+		return errors.Wrap(ErrChainID, "does not match the node's chainID")
```

Comment:
`return ErrChainID` or
`return errors.Wrap(ErrChainID, "chainID mismatch, expect %d, got %d")`

Reply:
and add test to cover this case